x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -1908,8 +1912,10 @@ describe('Task Runner', () => {
   });
 
   test('recovers gracefully when the Runner of a legacy Alert task which has no schedule throws an exception when fetching attributes', async () => {
-    rulesClient.get.mockImplementation(() => {
-      throw new Error(GENERIC_ERROR_MESSAGE);
```

Comment:
`rulesClient.get` is not used anywhere in the alerting TaskRunner, therefore I removed it from this test case as well as the some other tests below.

This was generating an error and passing because `encryptedSavedObjectsClient.getDecryptedAsInternalUser` mock was defined after `new TaskRunner()` definition.